backends/qualcomm/qnn_quantizer.py

```diff
@@ -158,15 +160,53 @@ def get_default_qnn_ptq_config(
     return quantization_config, quantizer_configs
 
 
+def get_16bit_qnn_ptq_config() -> Tuple[QuantizationConfig, QnnQuantizerConfig]:
+    extra_args: Dict[str, Any] = {"eps": 2**-20}
+    act_quantization_spec = QuantizationSpec(
+        dtype=torch.int32,
+        quant_min=0,
+        quant_max=65535,
+        qscheme=torch.per_tensor_affine,
+        observer_or_fake_quant_ctr=MovingAverageMinMaxObserver.with_args(**extra_args),
+    )
+
+    weight_quantization_spec = QuantizationSpec(
+        dtype=torch.int16,
+        quant_min=-32767,
+        quant_max=32767,
```

Comment:
is this int16 min/max? if so do you need to specify these?

Reply:
Yes, they are. For 16bit quantization, it seems that these need to be specified, otherwise the quant range seems to be wrong.

The following results are quant_attrs
![image](https://github.com/pytorch/executorch/assets/138087975/6200f6dd-92a6-458f-ae32-dbb49d50b08b)